Source/WebCore/history/BackForwardCache.cpp

```diff
@@ -355,43 +355,49 @@ void BackForwardCache::setMaxSize(unsigned maxSize)
     prune(PruningReason::None);
 }
 
-unsigned BackForwardCache::frameCount() const
+unsigned BackForwardCache::frameCount()
```

Comment:
I think instead of removing this const, we should add this to CachedPage:
    const CachedFrame* cachedMainFrame() const { return m_cachedMainFrame.get(); }